support-frontend/fetch-config.sh

```diff
@@ -8,5 +8,6 @@ if [[ -w ${CONFIG_DIR} ]]; then
     --profile membership
 else
   echo "ERROR! Cannot write to ${CONFIG_DIR}. Check it exists and you have write permissions to it.";
+  echo "  'mkdir ${CONFIG_DIR}' will create ${CONFIG_DIR} if it does not exist";
   echo "  'sudo chown -R $(whoami):admin ${CONFIG_DIR}' will make you the owner of ${CONFIG_DIR}";
```

Comment:
This can be scripted, for example, [here's how Grid](https://github.com/guardian/grid/blob/master/fetch-config.sh#L26-L30) handles something similar.

Reply:
i'm not confident enough in my shell chops to do this but happy to pair tomorrow!!